_posts/2019-04-29-opt-cuda-quantized.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  "Optimizing Quantized Operators on CUDA"
```

Comment:
Give the tile more context:

Automating Optimization of Quantized Deep Learning Models on CUDA